nvm.sh

```diff
@@ -2425,7 +2425,7 @@ nvm() {
           nvm_err '*** Third-party $NVM_INSTALL_THIRD_PARTY_HOOK env var claimed to succeed, but failed to install! ***'
           return 33
         fi
-        NVM_INSTALL_SUCCESS=true
+        NVM_INSTALL_EXIT_CODE=0
```

Comment:
i think this should set it to `"${EXIT_CODE}"` since that's the exit code from the hook.